index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./stylesheet/style.css">
+    <title>Document</title>
+    
+  </head>
+  <body>
+     <h1 class="text-center main-logo">My Library</h1>
+    <!-- <label for="title">Title</label>
+    <input type="text" name="title" id="title" />
+    <label for="author">author</label>
+    <input type="text" name="author" id="author" />
+    <label for="pages">pages</label>
+    <input type="text" name="pages" id="pages" />
+    <label for="read">read</label>
+    <input type="text" name="read" id="read" />
+    <button type="submit" id= 'create'>Create Book</button> -->

```

Comment:
. It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))